src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -122,25 +128,10 @@ full name
                 IsGlobal = false
             });
 
-            DocsSchema.DefineKey(new TableSchema.SchemaIndexDef
-            {
-                StartIndex = (int)DocumentsTable.LowerId,
-                Count = 1,
-                IsGlobal = true,
-                Name = DocsSlice
-            });
-            DocsSchema.DefineFixedSizeIndex(new TableSchema.FixedSizeSchemaIndexDef
-            {
-                StartIndex = (int)DocumentsTable.Etag,
-                IsGlobal = false,
-                Name = CollectionEtagsSlice
-            });
-            DocsSchema.DefineFixedSizeIndex(new TableSchema.FixedSizeSchemaIndexDef
-            {
-                StartIndex = (int)DocumentsTable.Etag,
-                IsGlobal = true,
-                Name = AllDocsEtagsSlice
-            });
+            DefineIndexesForDocsSchema(DocsSchema);
+            DefineIndexesForDocsSchema(CompressedDocsSchema);
+            DocsSchema.CompressValues(CompressedDocsSchema.FixedSizeIndexes[CollectionEtagsSlice], compress: false);
```

Comment:
```suggestion
            DocsSchema.CompressValues(DocsSchema.FixedSizeIndexes[CollectionEtagsSlice], compress: false);
```

Minor but shouldn't be this way?